api/swagger-spec/extensions.json

```diff
@@ -26,5 +26,83 @@
     ]
    }
   ],
-  "models": {}
+  "models": {
+   "unversioned.APIGroup": {
+    "id": "unversioned.APIGroup",
+    "description": "APIGroup contains the name, the supported versions, and the preferred version of a group.",
+    "required": [
+     "name",
+     "versions",
+     "serverAddressByClientCIDRs"
+    ],
+    "properties": {
+     "kind": {
+      "type": "string",
+      "description": "Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#types-kinds"
+     },
+     "apiVersion": {
```

Comment:
Server doesn't set the `apiVersion` field for this `unversioned.APIGroup` object at the endpoint /apis/extensions, so per @bgrant0607's comment, we shouldn't include the `apiversion` field in the swagger. But the same `unversioned.APIGroup` object is also in api/swagger-spec/batch.json and api/swagger-spec/autoscaling.json, where the `apiVersion` field should be included.
My suggestion is we expose the `apiVersion` field at all places to avoid confusion.